### PR TITLE
Added ability to disable automatic action caching.

### LINKED ISSUE #211: Allow disabling of cache
For my project, we are moving towards using S3 instead of Github cache so it would be useful to be able to disable the automatic caching.

### PR DESCRIPTION
**Pull request by rplankenhorn**
Fixes #211

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -12,6 +12,10 @@ inputs:
     required: false
     default: "true"
     description: 'if false, will not run `rtx install`'
+  enable_cache:
```
Comment (jdx) [inline action.yml:15]: just call it "cache"

File: src/cache-save.ts [hunk 2]
```diff
@@ -12,6 +12,13 @@ export async function run(): Promise<void> {
 }
 
 async function cacheRTXTools(): Promise<void> {
+  const skipCache = core.getState('SKIP_CACHE')
```
Comment (jdx) [inline src/cache-save.ts:15]: why not use core.getBooleanInput('enable_cache')?

Comment (rplankenhorn) [reply src/cache-save.ts:15]: I thought I remember reading somewhere that input aren't available in the post script but I can't find that article.  I also can't find any examples that call `getInput` from post scripts so this seemed safer.  I can change it if you have seen it work.

Comment (jdx) [reply src/cache-save.ts:15]: oh I see, well in that case let's just name it `CACHE` to match the config setting

**Pull request merged**